<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_CieDBCImportNodeSelectWidget</class>
 <widget class="QWidget" name="C_CieDBCImportNodeSelectWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import from DBC File - Node selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>27</number>
   </property>
   <property name="topMargin">
    <number>27</number>
   </property>
   <property name="rightMargin">
    <number>27</number>
   </property>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeGbx" name="pc_GroupBoxGeneral">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>13</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="stw_opensyde_gui_elements::C_OgeLabHeadingGroupBold" name="pc_LabelHeading">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="stw_opensyde_gui_elements::C_OgeFrameSeparator" name="pc_Frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>6</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_Information">
        <property name="horizontalSpacing">
         <number>12</number>
        </property>
        <item row="0" column="0">
         <widget class="stw_opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelNode">
          <property name="text">
           <string>Node</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="stw_opensyde_gui_elements::C_OgeCbxText" name="pc_ComboBoxDBCNode">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="stw_opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelMessages">
          <property name="text">
           <string>Messages:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="stw_opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelSignals">
          <property name="text">
           <string>Signals:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pc_HorizontalLayoutUserCancelOK">
     <item>
      <spacer name="pc_HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgePubCancel" name="pc_PushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgePubDialog" name="pc_PushButtonOK">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabHeadingGroupBold</class>
   <extends>QLabel</extends>
   <header>C_OgeLabHeadingGroupBold.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeFrameSeparator</class>
   <extends>QFrame</extends>
   <header>C_OgeFrameSeparator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubDialog</class>
   <extends>QPushButton</extends>
   <header>C_OgePubDialog.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeGbx</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbx.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubCancel</class>
   <extends>QPushButton</extends>
   <header>C_OgePubCancel.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGroupItem</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupItem.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeCbxText</class>
   <extends>QComboBox</extends>
   <header>C_OgeCbxText.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
